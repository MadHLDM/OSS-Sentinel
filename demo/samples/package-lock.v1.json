{
  "name": "demo",
  "lockfileVersion": 1,
  "dependencies": {
    "leftpad": {
      "version": "1.3.0",
      "dependencies": {
        "sublib": { "version": "0.1.0" }
      }
    },
    "uuid": { "version": "9.0.0" }
  }
}

