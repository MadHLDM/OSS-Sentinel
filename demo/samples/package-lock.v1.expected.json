[
  { "name": "leftpad", "version": "1.3.0", "ecosystem": "npm" },
  { "name": "sublib",  "version": "0.1.0", "ecosystem": "npm" },
  { "name": "uuid",    "version": "9.0.0", "ecosystem": "npm" }
]

